webpack.config.js

```diff
@@ -1,32 +1,32 @@
 const path = require('path');
- const HtmlWebpackPlugin = require('html-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
 
- module.exports = {
-   entry: {
-     index: './src/index.js',
-   },
-   devServer: {
+module.exports = {
+  entry: {
+    index: './src/index.js',
+  },
+  devServer: {
     static: './dist',
   },
```

Comment:
- [ ] [Optional] Please add `mode: 'development',` here in webpack config file to solve the warning that appears while running the dev server.

![image](https://user-images.githubusercontent.com/68796232/223507187-bb532606-c60b-401d-bd66-cc2e9a0649a8.png)